### DISCUSSION TITLE
How to create a client in a Typescript (ExpressJS) API?

### DISCUSSION THREAD

**Discussion by MikeVerlinde** [wevm/wagmi#1692] [Q&A] [answered]
I'd like to not only use Wagmi on my front-end, but also execute contractReads on the back-end using @wagmi/core. For this to work, I need to set-up a client. Where do I do this in an Express API?

**Comment by tmm** [comment, marked as answer]
You can call `createClient` in an entrypoint file (e.g. same as where you set up your express server).

**Reply by MikeVerlinde** [reply]
Works indeed, thank you :)
